publish.gradle

```diff
@@ -0,0 +1,127 @@
+apply plugin: 'maven-publish'
+apply plugin: 'signing'
+
+ext["signing.keyId"] = ''
+ext["signing.password"] = ''
+ext["signing.secretKeyRingFile"] = ''
+ext["ossrhUsername"] = ''
+ext["ossrhPassword"] = ''
+ext["sonatypeStagingProfileId"] = ''
+
+File secretPropsFile = project.rootProject.file('local.properties')
+if (secretPropsFile.exists()) {
+    Properties p = new Properties()
+    p.load(new FileInputStream(secretPropsFile))
+    p.each { name, value ->
+        ext[name] = value
+    }
+} else {
+    ext["signing.keyId"] = project.hasProperty('GPGSigningKeyID') ? "$GPGSigningKeyID" : ''
+    ext["signing.password"] = project.hasProperty('GPGSigningPassword') ? "$GPGSigningPassword" : ''
+    ext["signing.secretKeyRingFile"] = project.hasProperty('SigningSecretKeyRingFile') ? "$SigningSecretKeyRingFile" : ''
+    ext["ossrhUsername"] = project.hasProperty('OSSRHUsername') ? "$OSSRHUsername" : ''
+    ext["ossrhPassword"] = project.hasProperty('OSSRHPassword') ? "$OSSRHPassword" : ''
+    ext["sonatypeStagingProfileId"] = project.hasProperty('SonatypeStagingProfileID') ? "$SonatypeStagingProfileID" : ''
+}
+
+
+project.ext.bintrayPublishFunc = { bintrayContext, aarName ->
+    bintrayContext.user = project.hasProperty('bintrayUser') ? "$bintrayUser" : ""
+    bintrayContext.key = project.hasProperty('bintrayKey') ? "$bintrayKey" : ""
+    bintrayContext.publications = ['FluentUI']
+    bintrayContext.publish = true
+    bintrayContext.override = true
+    bintrayContext.pkg {
+        repo = 'generic'
+        name = aarName
+        version {
+            name = android.defaultConfig.versionName
+            vcsTag = android.defaultConfig.versionName
+        }
+    }
+}
+
+project.ext.publishingFunc = { artifactIdName ->
+    publishing {
+        repositories {
+            maven {
+                url 'https://pkgs.dev.azure.com/microsoftdesign/fluentui-native/_packaging/fluentui-android/maven/v1'
+                credentials {
+                    username = project.hasProperty("mavenUserName") ? "$mavenUserName" : ""
+                    password = project.hasProperty("mavenPassword") ? "$mavenPassword" : ""
+                }
+            }
+            maven {
+                name = "sonatype"
+                url = "https://s01.oss.sonatype.org/service/local/staging/deploy/maven2/"
+                credentials {
+                    username ossrhUsername
+                    password ossrhPassword
+                }
+            }
+        }
+        publications {
+            FluentUI(MavenPublication) {
+                groupId 'com.microsoft.fluentui'
+                artifactId artifactIdName
+                version = android.defaultConfig.versionName
+                artifact(sourceJar)
```

Comment:
Is there an override = true option here? Like in case of bintray?
When we change just one module, the version of library also is incremented (i.e the container module), but with publish function, publish of all modules is attempted hence for other unchanged modules, same version is published again

Reply:
This override = true is not supported, We will have to check this, once a module is published.